packages/styles/combobox.css

```diff
@@ -111,8 +111,24 @@
   position: relative;
 }
 
+.Combobox .TextFieldWrapper {
+  min-height: 32px;
+  flex-wrap: wrap;
+  gap: var(--space-half);
+  padding-inline: var(--space-smallest);
+  padding-block: var(--space-half);
```

Comment:
Is there a reason why these styles were added? This appears to change the size of all Comboboxes. Comboboxes should match the existing size of other comboboxes (which is currently 35px).